kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/UpdatedResourceSelection.vue

```diff
@@ -85,6 +85,15 @@
         type: Array,
         required: true,
       },
+      noSelectableResourcesIds: {
```

Comment:
Maybe to make what this is doing more explicit, this could be `alreadySelectedResourceIds` or `unselectableResourceIds`?

Reply:
I thought a lot about this, `alreadySelectedResourceIds` could be a little bit confusing because we can confuse it with the `selectedResources` that are the "working resources" of the lesson resource selection side panel.

I think I would go for `unselectabelResourceIds`, I think I didnt considered this option well because I confused this with `deselectableResourceIds`, but its actually not the same in english :laughing:.